### ISSUE TITLE
Consider changing (removing) QR code colours.

### ISSUE THREAD

**Issue by MikuroK** [uowuo/abaddon#297]
Hi, just wanted to point out that the "login with qr code" feature is more difficult to scan due to the colours used.

I have attached a picture demonstrating this by making a black/white version of the QR code. The barcode may be easier or harder scan depending on many factors such as scanning software, camera quality, the kind of screen it's displayed on et cetera, but intentionally reducing the contrast of the original image doesn't help.

![a](https://github.com/uowuo/abaddon/assets/1035669/a9a3fbaf-ff12-4359-abfd-40eb690cd8aa)

Thank you.

**Comment by Jaoheah** [comment]
Yep, I am having the same issue, the app is refusing to scan the qr

**Comment by Jaoheah** [comment]
It may not be the color that's causing the issue, in the discord app it's refusing to scan for me, but when I scan it normally with my phones own qr scanner it scans fine.

**Comment by MikuroK** [comment]
It's definitely the colour for me, though I don't doubt your experience either as performance could differ between the qr scanning app discord is using vs. your external app.
As demonstrated in the attached picture, a solid black/white version of the same code scans quickly and reliably. I also have two monitors, where the slightly older one I normally have discord open on I can't get it to scan on at all with the original purple colour, and only just on the other monitor if I hold my phone at a specific angle to the screen (LCD). it's not very convenient.
Maybe this doesn't affect many people, I'm scanning using a 2016 phone from a 2010 monitor, but it doesn't hurt anything to set it to the standard black/white colours, either.

**Comment by ouwou** [comment]
its black now
